src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -20,7 +20,7 @@ public class CookieAuthenticationHandler :
         IAuthenticationSignOutHandler
     {
         private const string HeaderValueNoCache = "no-cache";
-        private const string HeaderValueMinusOne = "-1";
+        private const string HeaderValueEpocDate = "Thu, 01 Jan 1970 00:00:00 GMT";
```

Comment:
Nit: why not just call this HeaderValueExpired so if you change this again, you don't have to rename it :)

Reply:
Also if this is only used in one place...these don't really even need to be consts do they?

Reply:
I'd rather have the name describe the value than where it gets used. It makes the usage code more obvious.
The constants also let you describe the value, and keep mics strings out of the code.